### PR TITLE
Responsive design bootstrap feature merge request

### PR DESCRIPTION
**Pull request by StarSheriff2**
For this project we cloned the NewsWeek website using bootstrap for most of the styling. We used Bootstrap's Grid system to make the page responsive at different breakpoints. We used 3 breakpoints. The Max-width of the page in desktop view is 1310px.

Only HTML and CSS besides Bootstrap were used in this feature.

### CODE REVIEW

File: index.html
Comment (descholar-ceo) [inline index.html]: [Optional] Don't load scripts in head, you should load this script at the bottom of the body before the closing tag of body (at the end of your html codes)

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Change Required :vertical_traffic_light: :warning: 
## Hello @StarSheriff2 and @Milypm  :handshake: :handshake: :handshake: 
:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.

### :warning: However we need to make few changes to mark this project approved. See the requested change below :point_down: :point_down: :point_down: 
## Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes]
- [x] Remove `!important` from your CSS codes, it's a bad practice over-using it
- [x] Fix in the magazine section on tablet screens (768px screens), the titles on bottom are overlapping

| Your clone |
| --- |
|Screenshot from 2020-11-11 08-42-34|
:point_right: Could you see the bottom of each sub-section :question: :question: :question: 

- [x] Implement all changes I suggested in inline comments in your codes

# [Optional changes]
- [x] Your commit messages should be in imperative mode or present tense, *i.e:* you should have witten `Style subscribe section on md breakpoint` instead of writting `styled subscribe section on md breakpoint`, refer [to this article for more details on how to write a good && professional commit message](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

If anything seems to be unclear to you, don't hesitate to contact me for more clarification slack: `@Mugirase Emmanuel` | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

File: index.html
Comment (StarSheriff2) [reply index.html]: done

**Review by PhillipUg — APPROVED** [review]
# Status: Project Approved! 🥇 

Congratulations on making all the requested changes and getting this project approved. :+1:

- [ ] Consider making the navbar fixed. so that it stays in place when you scroll down like on the real page.

- [x] The `Sports` section at the bottom just above the footer, has just one item to the right unlike other sections of its kind which have three items to the right. fix this

You can merge this branch into the main branch when you're ready and go on to your next project.
:bulb:  Do not forget to submit your project completion form after merging this branch.

All the best team,

For any assistance, you can contact me on slack as **Phillip Musiime**

**Pull request merged**
